euphonic/qpoint_phonon_modes.py

```diff
@@ -424,9 +424,9 @@ def calculate_pdos(
             self, dos_bins: Quantity,
             mode_widths: Optional[Quantity] = None,
             mode_widths_min: Quantity = Quantity(0.01, 'meV'),
-            adaptive_method: Optional[str] = 'reference',
+            adaptive_method: Literal['reference', 'fast'] = 'reference',
             adaptive_error: Optional[float] = 0.01,
-            adaptive_error_fit: str = 'cubic',
+            adaptive_error_fit: Literal['cheby-log', 'cubic'] = 'cubic',
```

Comment:
I did this for all the sampling schemes later, which get pretty unwieldy. I think for two cases this is fairly readable? But naming and re-using would reduce likelihood of error so maybe it's worth doing.

Reply:
I'm thinking more of maintainability in future if you're ever going add alternative methods, given I suspect this chains down functions it may work best to have a single point of contact. Possibly not worth it for the indirection?

Reply:
Done!